site/src/pages/[shortname]/blocks/[block-slug].page.tsx

```diff
@@ -85,6 +85,8 @@ const generateSandboxBaseUrl = (): string => {
   const branchSlug = branch
     .toLowerCase()
     .replace(/\./g, "")
+    .replace(/\//, "-")
+    .replace(/[/_]/g, "")
```

Comment:
Branch `dependabot/npm_and_yarn/next-12.2.4` maps to

`https://blockprotocol-git-dependabot-npmandyarnnext-1224.stage.hash.ai`

instead of

`https://blockprotocol-git-dependabot-npm_and_yarn-next-1224.stage.hash.ai`

Broken deployment example:
https://blockprotocol-npau13ws1.stage.hash.ai/@hash/blocks/code